config.json

```diff
@@ -201,6 +201,23 @@
                 "MAX_LIMIT": 2,
                 "WINDOW_SIZE": 1440
             }
+        },
+        {
+            "ID": "ECLIPSET",
+            "NAME": "Eclipse Testnet",
+            "TOKEN": "ECLP",
+            "RPC": "https://subnets.avax.network/eclipsecha/testnet/rpc",
+            "CHAINID": 555666,
+            "EXPLORER": "https://subnets-test.avax.network/eclipsecha",
+            "IMAGE": "https://raw.githubusercontent.com/Thor-Financial/Eclipse-assets/main/eclipse-logo.png",
+            "MAX_PRIORITY_FEE": "300000000000",
+            "MAX_FEE": "100000000000",
+            "DRIP_AMOUNT": 1000000000,
```

Comment:
Drip amount is in base units. So above would mean `1000000000 ECLP`. Is that expected?

Reply:
Yeah, the docs might be a little outdated. If you want 0.1 ECLP drip, then use 0.1

PS: Not necessary, but I would suggest keeping the drip amount integer.